Source/JavaScriptCore/parser/NodesAnalyzeModule.cpp

```diff
@@ -32,6 +32,23 @@
 
 namespace JSC {
 
+static RefPtr<ScriptFetchParameters> tryCreateAssertion(VM& vm, ImportAssertionListNode* assertionList)
+{
+    if (!assertionList)
+        return nullptr;
+
+    // Currently, only "type" is supported.
+    std::optional<ScriptFetchParameters::Type> type;
+    for (auto& [key, value] : assertionList->assertions()) {
+        if (*key == vm.propertyNames->type)
+            type = ScriptFetchParameters::parseType(value->impl());
```

Comment:
I guess we are to ignore unsupported keys here.

Reply:
Yes, right now, we are hard-coding it because it is still not sure whether we will have platform-specific supported keys (so, not sure whether we need extensibility here). For now, keeping it, and later, if we start having yet another assertions, then, let's revisit. (I wonder if we end up always having the same supported assertions for every platforms JSC support, and probably it is fine).